neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -26,122 +28,9 @@ public sealed class NeoToken : Nep5Token<NeoToken.AccountState>
         private const byte Prefix_ValidatorsCount = 15;
         private const byte Prefix_NextValidators = 14;
 
-        internal NeoToken() : base()
+        internal NeoToken()
```

Comment:
We should call the base constructor
```suggestion
        internal NeoToken() : base()
```

Reply:
In C#, base constructor is automatically called by subclass.